collections/eurocrops.yaml

```diff
@@ -0,0 +1,112 @@
+  
+  This work is licensed under a [Creative Commons Attribution 4.0 International License](http://creativecommons.org/licenses/by/4.0/).
+  
+  An algorithm for retrieval of [Eurocrops](https://github.com/maja601/EuroCrops) Sentinel-2 signals based on an area of interest.
+  
+  The algorithm allows the users to specify:
+    * an area-of-interest (AoI);
+    * a time interval for which they want to retrieve Sentinel-2 data;
+    * an evalscript specifying which Sentinel-2 bands or derived indices, e.g. NDVI, to download.
+  
+  Provided the above parameters, the algorithm computes Sentinel-2 time-series for the specified bands/indices for each polygon in the Eurocrops dataset that intersects the AoI. The computed signals are aggregated spatially over the polygon, and temporally over one day. 
+  
+  Such signals constitute the basis for many applications, in particular in monitoring agricultural activity. Please refer to this [blog-post series](https://medium.com/sentinel-hub/area-monitoring-concept-effc2c262583) for examples on how these signals can be used.
+  
+  The tool relies on AWS S3 bucket to store the results, thus read/write access to an AWS S3 bucket is required. 
+  
+  The process consists of the following steps:
+    - Retrieve Eurocrops parcels for a chosen AoI. This is done with the help of [GeoDB](https://github.com/dcs4cop/xcube-geodb).
+    - The parcels are saved to the specified folder on the user's AWS S3 bucket. 
+    - Retrieve signals for the parcels in the AoI for the specified time interval using Sentinel Hub [Batch Statistical API](https://docs.sentinel-hub.com/api/latest/api/batch-statistical/).
+    - The signals are downloaded to the  user's AWS S3 bucket in the form of `.json` files.
+Image: eurocrops/thumbnail.png
+Explore: "[Run algorithm in EDC Browser](https://browser.eurodatacube.com/?zoom=10&lat=41.9&lng=12.5&fromTime=1970-01-01T00%3A00%3A00.000Z&toTime=2022-11-14T13%3A45%3A29.045Z&algorithm=eurocrops)"
+GeographicalCoverage: |
+  - Austria
+  - Belgium
+  - Germany
+  - Denmark
+  - Estonia
+  - Spain
+  - France
+  - Croatia
+  - Lithuania
+  - Latvia
+  - Netherlands
+  - Portugal
+  - Romania
+  - Sweden
+  - Slovenia
+  - Slovakia
+TemporalAvailability: |
+  - Austria: 2021
+  - Belgium: 2021
+  - Germany: 2021
+  - Denmark: 2019
+  - Estonia: 2021
+  - Spain: 2020
+  - France: 2018
+  - Croatia: 2020
+  - Lithuania: 2021
+  - Latvia: 2021
+  - Netherlands: 2021
+  - Portugal: 2021
+  - Romania: Data is collected across different years.
+  - Sweden: 2021
+  - Slovenia: 2021
+  - Slovakia: 2021
+  
+UpdateFrequency: On-demand
+Attributes:
+  Table:
+    Columns:
+      - Name: Name
+        Title: Name
+      - Name: Description
+        Title: Description
+      - Name: Type
+        Title: Type
+    Rows:
+      - Name: aoi
+        Description: Area of interest
+        Type: Polyon or bounding box
```

Comment:
I missed this earlier sorry:) Just a small detail 

Adapt the attributes name to be exactly as it appears in EDC browser interface and also the order it appears : i.e 
- Area of Interest
- Time range
- Bucket Name
- Project folder
- AWS access key ID
- AWS secret access key
- Evalscript path

Reply:
Fixed. Btw I checked [sice.yaml](https://github.com/eurodatacube/public-collections/blob/main/collections/sice.yaml) and [field-delineation.yaml](https://github.com/eurodatacube/public-collections/blob/main/collections/field-delineation.yaml) and it seems to me that the attribute name doesn't need to be the same as what is shown on EDC Browser.

Reply:
Thanks. everything works now. 

Merging